Library/Homebrew/cask/audit.rb

```diff
@@ -583,7 +583,7 @@ def check_livecheck_min_os
       return if cask_min_os == min_os_string
 
       add_error "Upstream defined #{min_os_string.to_sym.inspect} as minimal OS version " \
-                "and the cask defined #{cask_min_os.to_sym.inspect}"
+                "and the cask defined #{cask_min_os ? cask_min_.to_sym.inspect : "no minimal OS version"}."
```

Comment:
How about this:

```ruby
min_os_symbol = cask_min_os&.to_sym.inspect || "no minimal OS version"
add_error "Upstream defined #{min_os_string.to_sym.inspect} as minimal OS version " \
          "and the cask defined #{min_os_symbol}."
```

I find that a bit easier to read

Reply:
@Rylan12 Looks good to me. I've updated it, thanks.